src/main/java/com/thealgorithms/backtracking/MazeRecursion.java

```diff
@@ -107,14 +105,14 @@ public static boolean setWay(int[][] map, int i, int j) {
                 return true;
             } else {
                 // means that the current point is the dead end, the ball cannot proceed, set
-                // the current point to 3 and return false, the backtraking will start, it will
+                // the current point to 3 and return false, the backtracking will start, it will
                 // go to the previous step and check for feasible path again
                 map[i][j] = 3;
                 return false;
             }
         } else { // if the map[i][j] != 0 , it will probably be 1,2,3, return false because the
             // ball cannot hit the wall, cannot go to the path that has gone though before,
-            // and cannot head to deadend.
+            // and cannot head to deadened.
```

Comment:
I think this correction is wrong. Please justify the change

Reply:
i will correct it